### ISSUE TITLE
[Question] Development Status

### ISSUE THREAD

**Issue by alkaris2** [derrod/lg4k-linux#17]
Is this still being developed on or no? I see this hasn't been updated in 2 Years..

**Comment by derrod** [comment]
Not really, I don't have the time and don't run Linux as my main operating system right now. Maybe I'll come back to it some day, don't know. For now there are some forks that have been updated to work with newer kernels and features, but are ultimately also unmaintained.
